examples/flight-management/backend/README.md

```diff
@@ -0,0 +1,17 @@
+This is a [Glee](https://github.com/asyncapi/glee) project bootstrapped with [`create-glee-app`](https://github.com/asyncapi/create-glee-app).
```

Comment:
I'd remove this file because it mentions `create-glee-app` and we want to get rid of it. Or at least remove mentions to `create-glee-app`.